llama_stack/distribution/ui/page/playground/tools.py

```diff
@@ -133,7 +133,8 @@ def response_generator(turn_response):
                             yield response.event.payload.delta.text
                     if response.event.payload.event_type == "step_complete":
                         if response.event.payload.step_details.step_type == "tool_execution":
-                            yield " 🛠 "
+                            tool_name = str(response.event.payload.step_details.tool_calls[0].tool_name)
```

Comment:
is `response.event.payload.step_details.tool_calls[0]` safe?

Reply:
I _think_ it should be, but good catch. Let me add a check here just to prevent any edge cases where the list might be empty.